zuul-netflix/src/main/java/com/netflix/zuul/dependency/ribbon/hystrix/RibbonCommand.java

```diff
@@ -53,13 +61,27 @@ public RibbonCommand(String commandKey,
                          MultivaluedMap<String, String> params,
                          InputStream requestEntity) throws URISyntaxException {
 
-        super(Setter.withGroupKey(HystrixCommandGroupKey.Factory.asKey(commandKey)).andCommandPropertiesDefaults(
-                // we want to default to semaphore-isolation since this wraps
-                // 2 others commands that are already thread isolated
-                HystrixCommandProperties.Setter().withExecutionIsolationStrategy(HystrixCommandProperties.ExecutionIsolationStrategy.SEMAPHORE)
-                        .withExecutionIsolationSemaphoreMaxConcurrentRequests(DynamicPropertyFactory.getInstance().
-                                getIntProperty(ZuulConstants.ZUUL_EUREKA + commandKey + ".semaphore.maxSemaphores", 100).get())));
-
+        // Switch the command/group key to remain passive with the previous release which used the command key as the group key
+        this(commandKey, RibbonCommand.class.getSimpleName(), restClient, verb, uri, headers, params, requestEntity);
+    }
+    
+    public RibbonCommand(String groupKey, 
+                    String commandKey,
+                    RestClient restClient,
+                    Verb verb,
+                    String uri,
+                    MultivaluedMap<String, String> headers,
+                    MultivaluedMap<String, String> params,
+                    InputStream requestEntity) throws URISyntaxException {
+        
+        super(Setter.withGroupKey(HystrixCommandGroupKey.Factory.asKey(groupKey))
+                    .andCommandKey(HystrixCommandKey.Factory.asKey(commandKey)).andCommandPropertiesDefaults(
+           // we want to default to semaphore-isolation since this wraps
+           // 2 others commands that are already thread isolated
+           HystrixCommandProperties.Setter().withExecutionIsolationStrategy(HystrixCommandProperties.ExecutionIsolationStrategy.SEMAPHORE)
+                   .withExecutionIsolationSemaphoreMaxConcurrentRequests(DynamicPropertyFactory.getInstance().
+                           getIntProperty(ZuulConstants.ZUUL_EUREKA + commandKey + ".semaphore.maxSemaphores", 100).get())));
```

Comment:
I wasn't sure what to do about the semaphore properties, it seems like this should be using the group key.